.github/workflows/pr-jobs.yml

```diff
@@ -1,3 +1,12 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
 name: CI
```

Comment:
Should we change the name of this now that this file was renamed from main -> pr-jobs?

Reply:
Good point, I made it more clear